examples/ksml-runner.yaml

```diff
@@ -30,38 +92,37 @@ ksml:
 # These examples are intended to run from a inside a container on the same network
 kafka:
   application.id: io.ksml.example.streaming
-#  bootstrap.servers: localhost:9092
-#  schema.registry.url: http://localhost:8081
   bootstrap.servers: broker:9093
-  schema.registry.url: http://schema_registry:8081
+  schema.registry.url: http://localhost:8081
```

Comment:
The examples are run in a docker container, and URLs should be schema_registry

Reply:
Corrected